src/state_machine.rs

```diff
@@ -188,6 +169,14 @@ impl State {
             state.set_next_node_name(relocation_name);
         }
     }
+
+    pub fn poll(&mut self) -> Vec<u64> {
```

Comment:
pls see the review at https://github.com/maidsafe/routing/pull/1438/files#diff-c9f7984a0a4f0ccaf63472cce67a9e67R3152